package.json

```diff
@@ -24,9 +24,10 @@
     "lint-sol": "solhint ./contracts/**/*.sol",
     "plant": "npx node-plantuml ./docs/specs",
     "prepare": "husky install",
+    "storage": "hardhat storage-layout --details",
     "sizes": "hardhat contract-sizes --details --diff --size 20000",
-    "sizes:all": "hardhat contract-sizes --diff",
-    "test": "mocha --timeout 10000 --exit --recursive --require ts-node/register \"test/**/*.test.ts\""
```

Comment:
question: are you dropping the test target?

Reply:
no, it is the next line - I've just added a comma here